frontend/src/components/pages/UserProfilePage.tsx

```diff
@@ -1,18 +1,162 @@
+
+  /* eslint-disable */
+  const [approvedLanguagesTranslation, setApprovedLanguagesTranslation] =
+    useState<{ [name: string]: number }>();
+  const [approvedLanguagesReview, setApprovedLanguagesReview] =
+    useState<{ [name: string]: number }>();
+
+  const history = useHistory();
+
+  useQuery(GET_USER(parseInt(userId, 10)), {
+    fetchPolicy: "cache-and-network",
+    onCompleted: (data: { userById: User }) => {
+      const user = data.userById;
+      const translationLanguages = user?.approvedLanguagesTranslation
+        ? JSON.parse(user.approvedLanguagesTranslation.replace(/'/g, '"'))
+        : {};
+      const reviewLanguages = user?.approvedLanguagesReview
+        ? JSON.parse(user.approvedLanguagesReview.replace(/'/g, '"'))
+        : {};
+      setFullName(`${user?.firstName} ${user?.lastName}`);
+      setEmail(user?.email!);
+      setApprovedLanguagesTranslation(translationLanguages);
+      setApprovedLanguagesReview(reviewLanguages);
+      setRole(
+        `Translator ${
+          Object.keys(reviewLanguages).length > 0 ? "& Reviewer" : ""
+        }`,
+      );
+    },
+    onError: () => {
+      history.push("/404");
+    },
+  });
+
+  const [deleteUser] = useMutation<{
+    response: SoftDeleteUserResponse;
+  }>(SOFT_DELETE_USER);
+
+  const closeModal = () => {
+    setConfirmDeleteUser(false);
+  };
+
+  const openModal = () => {
+    setConfirmDeleteUser(true);
+  };
+
+  const callSoftDeleteUserMutation = async () => {
+    await deleteUser({
+      variables: {
+        id: userId,
+      },
+    });
+    closeModal();
+    history.push("/");
+  };
+
+  const filterStyle = useStyleConfig("Filter");
   return (
-    <Box textAlign="center">
-      <h1>User Profile for {userId} :)</h1>
-    </Box>
+    <Flex direction="column" height="100vh">
+      <Header />
+      <Flex direction="row" flex={1}>
+        <Flex sx={filterStyle} maxWidth="400px">
+          <Heading size="lg">{fullName}</Heading>
+          <Heading size="sm" marginTop="36px">Role(s)</Heading>
+          <Text>{role}</Text>
+          <Heading size="sm" marginTop="36px">Email</Heading>
+          <Text>{email}</Text>
+          <Heading size="sm" marginTop="36px">Date joined</Heading>
+          <Text>TODO</Text>
+          <Heading size="sm" marginTop="36px">Last active on</Heading>
+          <Text>TODO</Text>
+        </Flex>
+        <Flex direction="column" margin="40px">
+          <Heading size="lg" marginTop="40px">
+            Approved Languages and Levels
+          </Heading>
+          <Text>TODO: table here</Text>
+          <Heading size="lg" marginTop="56px">
+            Assigned Story Translations
+          </Heading>
+          <Text>TODO: table here</Text>
+          <Heading size="sm" marginTop="56px">
+            Delete user
+          </Heading>
+          <Text>
+            Permanently delete this user and all data associated with them.
+          </Text>
+          <Button
```

Comment:
nit: On Figma it appears that this button is white with a red outline and text instead of red with white text. Also, maybe make it a little less wide and add some more top margin (very minor nit, please use your own judgment).

Reply:
I went with this for now in managestorytranslationpage as we're having problems with the `outline` variant.  so this is 👍